### DISCUSSION TITLE
update/giscus-comments/

### DISCUSSION THREAD

**Discussion by giscus** [OpenGammaProject/Gamma-Spectrum-Database#7] [Announcements]
# update/giscus-comments/

I have finally looked into how to set up a commenting feature for some pages and here I am now.

https://gammadb.nuclearphoenix.xyz/update/giscus-comments/

**Comment by NuclearPhoenixx** [comment]
Welcome to the comments! And before you ask: No, this website is not dead as you can see ;)
